### PR TITLE
evscript: unstable-2022-11-20 -> 0.1.0

### PR DESCRIPTION
**Pull request by peterhoeg**
## Description of changes

First proper release - works swimmingly!

## Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [X] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/inputmethods/evscript/default.nix [hunk 1]
```diff
@@ -2,22 +2,17 @@
...
+    hash = "sha256-lCXDDLovUb5aSOPTyVJL25v1JT1BGrrUlUR0Mu0XX4Q=";
   };
 
-  cargoLock = {
-    lockFile = ./Cargo.lock;
-    outputHashes = {
-      "evdev-0.10.1" = "sha256-iIzKhlA+7qg+nNwP82OIpoXVUEYU31iSEt1KJA3EewQ=";
-    };
-  };
+  cargoLock.lockFile = ./Cargo.lock;
```
Comment (alyssais) [inline pkgs/tools/inputmethods/evscript/default.nix:15]: If there are no git dependencies, we don't commit the Cargo.lock in Nixpkgs atm.

**Comment by peterhoeg** [comment]
will squash on merge

**Comment by peterhoeg** [comment]
any concerns @MilesBreslin ?

**Review by MilesBreslin — APPROVED** [review]
Looks good to me! Thanks!

**Comment by amjoseph-nixpkgs** [comment]
Was this an accidental commit?

```
commit 3d7a169afe725339aeb7e24b8407fd8a7a013316 (origin/master)
Author: Peter Hoeg <peter@hoeg.com>
Date:   Thu Oct 19 11:26:07 2023 +0200

    WIP

diff --git a/pkgs/tools/inputmethods/evscript/Cargo.lock b/pkgs/tools/inputmethods/evscript/Cargo.lock
deleted file mode 100644
index 7f977826b101..000000000000
--- a/pkgs/tools/inputmethods/evscript/Cargo.lock
+++ /dev/null
```

**Comment by peterhoeg** [comment]
I was supposed to have squashed the commits. Note to self - next time squash immediately.

**Pull request merged**